nexus/src/app/mod.rs

```diff
@@ -477,6 +482,27 @@ impl Nexus {
             }
         });
 
+        // Spawn a task to receive SagaRequests from RPWs, and execute them
+        {
+            let nexus = nexus.clone();
+            tokio::spawn(async move {
+                loop {
+                    match saga_request_recv.recv().await {
+                        None => {
+                            // If this channel is closed, then RPWs will not be
+                            // able to request that sagas be run. Panic Nexus so
+                            // that it can be restarted.
+                            panic!("saga request channel closed!");
+                        }
```

Comment:
Could this be a signal that we should shut down? Might be better to "log a warning, then exit" rather than panicking.

Reply:
This is a good point, changed to `error!` in b2be97f6e